### ISSUE TITLE
Example of File API

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#868]
```
Based on the suggestions from the existing File API article 
(http://www.html5rocks.com/en/tutorials/file/filesystem/), I've got an example 
that makes use of the API to...

1) Conditionally download a large zip (based on checking if the zip has been 
updated)
2) Makes use of a JavaScript Zip library to extract contents
3) Copies contents to local file system
4) Then makes use of the files.

Basically the idea is to show a real world use of the File System API.
```

Original issue reported on code.google.com by `raymondc...@gmail.com` on 10 Apr 2012 at 1:49

**Comment by GoogleCodeExporter** [comment]
```
My demo here does the zip extraction part:
http://www.htmlfivewow.com/demos/photo-gallery/index.html

```

Original comment by `ericbide...@html5rocks.com` on 16 Apr 2012 at 5:44

**Comment by GoogleCodeExporter** [comment]
```
It was my understanding that we should use the issue tracker to record things 
we would like to write about on the site. I'm aware of demos out there using 
zip and the file system API, but I had wanted to actually write something up 
for the site. Is that not the proper user then for this?
```

Original comment by `raymondc...@gmail.com` on 16 Apr 2012 at 6:10

**Comment by GoogleCodeExporter** [comment]
```
Ah sorry, I didn't realize this is for a tutorial.

...Raymond contacted me directly and think he's already written this article 
for another site.
Marking this as WontFix.
```

Original comment by `ericbidelman@chromium.org` on 25 Apr 2012 at 10:55
- Changed state: **WontFix**
